lib/Component.php

```diff
@@ -164,6 +164,21 @@ public function render($params = array()) {
       }
     }
 
+    // Check for translations. This differs a bit from ds-erlang since there all translations
+    // are chucked into the root component, i.e. the theme. Here we render one component and
+    // don't support children so we let the component have it's own translations.
```

Comment:
Well, there IS support for subcomponents, tested with complete theme-rendering, in https://github.com/DiversityTemplating/diversity-php/pull/10 :)  …but that was a while ago…